src/main/scala/sigmastate/utxo/transformers.scala

```diff
@@ -414,3 +415,31 @@ case class DeserializeRegister[V <: SType](reg: RegisterId, tpe: V, default: Opt
   override val opCode: OpCode = OpCodes.DeserializeRegisterCode
   override def cost[C <: Context[C]](context: C): Long = 1000 //todo: rework, consider limits
 }
+
+case class GetVar[V <: SType](varId: Byte, override val tpe: SOption[V]) extends NotReadyValue[SOption[V]] {
+  override val opCode: OpCode = OpCodes.GetVarCode
+  override def cost[C <: Context[C]](context: C): Long = context.extension.cost(varId) + 1
+}
+
+object GetVar {
+  def apply[V <: SType](varId: Byte, innerTpe: V): GetVar[V] = GetVar[V](varId, SOption(innerTpe))
+
+  def GetVarBoolean(varId: Byte): GetVar[SBoolean.type] = GetVar(varId, SBoolean)
```

Comment:
@ergomorphic Do you think it's these methods better fit in `Values` object along with `TaggedInt` and friends? This would save one import line, although I think they make more sense here.

Reply:
@greenhat Yes, probably, `GetVar.GetVarBoolean(10)` looks too verbose.